src/main/java/org/folio/dew/service/SaveErrorService.java

```diff
@@ -0,0 +1,39 @@
+package org.folio.dew.service;
+
+import static java.util.Objects.isNull;
+import lombok.extern.log4j.Log4j2;
+import org.springframework.stereotype.Service;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+@Service
+@Log4j2
+public class SaveErrorService {
```

Comment:
According to the initial design csv-file should be saved on S3 and the only link to download will be provided to end user? @IlliaDaliek , please, confirm.

Reply:
As per the last comment in https://issues.folio.org/browse/MODBULKED-10, we do not need to save error file in S3.

Reply:
This case all the error fils will be saved on local FS. I don't see any mechanism to remove these files and clear space.

Reply:
After discussion at the meeting we decided to use S3 to store file with errors